package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "eslint-config-foxsports",
+  "version": "0.0.0",
```

Comment:
The [semver 2.0 spec](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) recommends starting at 0.1.0